python/pyspark/sql/dataframe.py

```diff
@@ -4066,6 +4078,9 @@ def dropDuplicatesWithinWatermark(self, subset: Optional[List[str]] = None) -> "
 
          .. versionadded:: 3.5.0
 
+        .. versionchanged:: 3.5.0
```

Comment:
Why would a user need to be notified of the change since it was added in version 3.5.0?

Reply:
Yeah it's a bit odd but we have used this way to describe the support of Spark Connect. Let's keep it consistent for now, and fix all later separately.

Reply:
yes, this PR is just to following existing style